deploy/docker/tests/test-pg-auth.sh

```
@@ -108,6 +108,26 @@ check_user_datasource_access_with_local_port_wo_auth() {
   return $?
 }
 
+# Function to check if the user exists in the database
+check_user_exists() {
+  local user
+  user=$1
+  local user_exists
```

Shellcheck (0.10.0)

[warning] user_exists appears unused. Verify use (or export if used externally).

(SC2034)